requirements.txt

```diff
@@ -1 +1 @@
-django==1.8.15
\ No newline at end of file
+django>=1.8.15,<1.12
```

Comment:
(I'm still learning how python projects work, so forgive me if this is a well-answered question elsewhere). 

[This documentation](https://github.com/cfpb/development/blob/master/standards/python.md#requirementstxt-setuppy-and-specifying-dependencies) specifies that, since we use satellite apps like libraries, we should specify their dependencies in `setup.py` instead of in a `requirements.txt`. 

I know you didn't create this requirements.txt file in the first place, so it wasn't necessarily your choice, but I'm curious: are there pros and cons to the different approaches that we should consider?

Reply:
I think it's a great question. In this particular case I'm not too sure what the pros and cons would be in practice, and I don't feel too strongly either way. So in that case, let's see what @chosak thinks. I'm glad you brought this up 👍

Reply:
My current preference is to put these dependencies in the `install_requires` in setup.py, and avoid using requirements.txt in satellite apps.

This allows users of this package to do `pip install ccdb5-ui` and have all of the necessary dependencies brought in. As written, the `setup.py` in this package doesn't bring in Django; that means in practice you can't run the tests unless you also install Django manually, or use Tox.

Consider a case where someone is using Django 2.0, and wants to pip install this package. Ideally pip should tell us that it isn't yet compatible with that version of Django, and fail -- that would happen if setup.py contained the constraints.

Using setup.py is nice because if you have a slightly older version of a supported dependency installed first, say, Django 1.11.3, the `install_requires` will match, and your existing version won't be uninstalled/modified. If you don't already have the dependency installed, then the latest compatible version will be installed. Either way you're guaranteed to have a working set of packages.

There are probably some advantages to requirements.txt I'm not thinking of -- for one, it's slightly easier to specify working branches or releases on GitHub (a single line like [this](https://github.com/cfpb/cfgov-refresh/blob/master/requirements/optional-public.txt#L8)) instead of using PyPI installs. [Here's](https://stackoverflow.com/a/20234833) how you have to do it in setup.py.

Reply:
Thanks @chosak for the excellent thoughts. Would you and @schbetsy suggest I make this change in the present PR? Or open an issue – ?

Reply:
My vote would be to include in this PR.

Reply:
Thanks for the great write-up @chosak . That's super helpful.